example/MyFirstDrawing/MyFirstDrawingUITests/BrushToolView/GroupEraseSizeViewUITests.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  GroupEraseSizeViewUITests.swift
+//  MyFirstDrawingUITests
+//
+//  Created by Stant Macmini n04 on 09/07/19.
+//  Copyright © 2019 Stant 02. All rights reserved.
+//
+
+import XCTest
+import StantDrawing
+@testable import MyFirstDrawing
+
+class GroupEraseSizeViewUITests: XCTestCase {
+    let app = XCUIApplication()
+    
+    override func setUp() {
+        continueAfterFailure = false
+        
+        app.launch()
+        
+        let btnErase = app.buttons["eraserfullIdentifier"]
+        btnErase.tap()
+    }
+    
+    override func tearDown() {}
+    
+    func testClickOnGroupEraseSizeViewAndShowAllButtons() {
+        let eraseSizeReference = app.images.matching(identifier: "eraserfullReferenceIdentifier")
+        
+        let btnEraseSize1 = app.buttons["eraserSize1Identifier"]
+        let btnEraseSize2 = app.buttons["eraserSize2Identifier"]
+        let btnEraseSize3 = app.buttons["eraserSize3Identifier"]
+        let btnEraseSize4 = app.buttons["eraserSize4Identifier"]
```

Comment:
Anyway, avoid btn uses instead of button.

Reply:
Done